dags/benk/prepare_config/update.py

```diff
@@ -0,0 +1,34 @@
+"""To update the prepare definitions in this directory, run from 'dags' directory.
+
+python -m benk.prepare_config.update
+
+"""
+
+from pathlib import Path
+
+import requests
+
+from benk.definitions import DEFINITIONS
+
+GITHUB_BASE_URL = "https://raw.githubusercontent.com/Amsterdam/GOB-Prepare/master"
+
```

Comment:
mss op te lossen met git submodule voor sync?

Reply:
Good suggestion. Did just that now :)

Reply:
I hoped to auto sync the definitions from the prepare repo, but don't know if the git sync is recursive..

Reply:
Not a big fan of git sync here, too much magic going on.

However; I think it would be nice to implement an automatic action (Github Action?) that updates this repo automatically whenever GOB-Prepare has an update.

Reply:
allright 👍 
Will try to test this locally, should be possible though